.github/workflows/release-e2e.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: release-e2e
```

Comment:
Follow-up: update this workflow to automatically create a github issue whenever the job fails, e.g. we fail to successfully install the release, or e2e tests fail, etc.).